src/main/java/com/aws/greengrass/util/ProxyUtils.java

```diff
@@ -286,7 +286,14 @@ public static SdkHttpClient getSdkHttpClient() {
                 certificates.addAll(EncryptionUtils.loadX509Certificates(Paths.get(rootCAPath)));
             }
 
+#if ANDROID
+
+            // NOTE: android: does not support "JKS"
+            KeyStore customKeyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+#else
             KeyStore customKeyStore = KeyStore.getInstance("JKS");
+#endif
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;Invalid Character&#34;, @ expected

This file contains syntax errors and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors. Fix the syntax errors and run the repository analysis or submit the pull request again.